app/redux-modules/alerts.js

```diff
@@ -212,26 +167,25 @@ export function resetAlertsDb() {
 
 
 // Action Creators
-export function setCanDisplayAlerts(canDisplay) {
+export function setCanDisplayAlerts(canDisplay: boolean) {
   return {
     type: SET_CAN_DISPLAY_ALERTS,
     payload: canDisplay
   };
 }
 
 export function setActiveAlerts() {
-  return (dispatch, state) => {
+  return (dispatch: Dispatch, state: GetState) => {
     const areas = state().areas;
     const index = areas.selectedIndex;
     const area = areas.data[index] || null;
     const dataset = activeDataset(area);
-    const datasetSlug = dataset && dataset.slug;
     const canDisplay = state().alerts.canDisplayAlerts;
 
-    const action = { type: SET_ACTIVE_ALERTS, payload: null };
-    if (datasetSlug && canDisplay) {
-      activeCluster.supercluster = memoizedAreaToClusters(area.id, datasetSlug, dataset.startDate);
-      action.payload = `${area.id}_${datasetSlug}_${dataset.startDate}`;
+    const action = ({ type: SET_ACTIVE_ALERTS, payload: null }: { type: string, payload: ?string });
+    if (dataset && canDisplay) {
+      activeCluster.supercluster = memoizedAreaToClusters(area.id, dataset.slug, dataset.startDate);
```

Comment:
Let's clear the cache when the alerts are updated. I just realised this is not the case same as LOGOUT_REQUEST.

Reply:
👀